doc/visual-programming/source/widgets/data/datainfo.rst

```diff
@@ -26,23 +26,23 @@ Description
 -----------
 
 A simple widget that presents information on data set size, features,
-targets, meta attributes, and location. It is useful when you manually
-select a subset and you want to see information on the selected subset.
+targets, meta attributes, and location. 
 
-.. figure:: images/data-info4-stamped.png
+.. figure:: images/data-info-stamped.png
 
 1. Information on data set size
-2. Information on discrete and continuous features
+2. Information on discrete and numeric features
```

Comment:
"Continuous" is rooted in Orange code. For documentation, it is ok to use both, but you can use "numeric" through documentation and at the first mention tell in brackets that that is the same as "continuous".